sale_commission_product_criteria/models/commission.py

```diff
@@ -0,0 +1,275 @@
+# © 2023 ooops404
+# License AGPL-3 - See https://www.gnu.org/licenses/agpl-3.0.html
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools import float_repr
+
+
+class SaleCommission(models.Model):
+    _inherit = "sale.commission"
+
+    commission_type = fields.Selection(
+        selection_add=[("product", "Product criteria")],
+        ondelete={"product": "set default"},
+    )
+    item_ids = fields.One2many("commission.item", "commission_id", copy=True)
+    sequence = fields.Integer(
```

Comment:
This sequence should go out as alien feature.

Reply:
updated